src/ssl.c

```diff
@@ -11021,6 +11099,154 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
         return SSL_NOT_IMPLEMENTED;
     }
 
+    #if defined(WOLFCRYPT_HAVE_SRP) && !defined(NO_SHA256) \
+    && !defined(WC_NO_RNG)
+    const byte N[] = {
```

Comment:
Please make this a `static const byte`. The static is important here so the symbol (variable name) is only available in ssl.c. Please rename to something unique that won't ever conflict like `srp_N`, like you did for `srp_g`.

Reply:
Yes,it will cause variable name conflict with other file.
I changed variable name from N to srp_N.
And I added static at srp_N and srp_g declaration.